src/components/Simulation/list.js

```diff
@@ -39,118 +38,100 @@ const mapOrgunits = invoices => {
   return uniqWith(all, (a, b) => a.key === b.key);
 };
 
-class SimulationList extends React.Component {
-  constructor(props) {
-    super(props);
-    const periods = mapPeriods(props.simulations);
-    const packages = mapPackages(props.simulations);
-    const orgunits = mapOrgunits(props.simulations);
-    this.state = {
-      periods,
-      allPeriods: periods,
-      packages,
-      allPackages: packages,
-      orgUnits: [orgunits[0]],
-      allOrgUnits: orgunits,
-    };
+const SimulationList = props => {
+  const { simulations } = props;
+  const allPeriods = mapPeriods(simulations);
+  const allPackages = mapPackages(simulations);
+  const allOrgUnits = mapOrgunits(simulations);
+
+  const [periods, setPeriods] = useState([]);
+  const [packages, setPackages] = useState([]);
+  const [orgUnits, setOrgUnits] = useState([]);
+
+  // Set default selection
+  if (orgUnits.length < 1 && allOrgUnits.length > 0) {
+    setOrgUnits([allOrgUnits[0]]);
   }
+  if (periods.length < 1 && allPeriods.length > 0) setPeriods(allPeriods);
+  if (packages.length < 1 && allPackages.length > 0) setPackages(allPackages);
+
+  const filteredSimulations = simulations.filter(simulation => {
```

Comment:
Technically should happen in the SimulationContainer. But you maybe wanted to do that in a subsequent PR

Reply:
If this file is the container, it shall be moved and striped from html ^^

Reply:
Yes, want to fix that in an upcoming PR (which also puts the filters on the right side in the sidesheet)

Reply:
But this does raise the question, is there a naming convention for loaders vs managers of data? Like the `SimulationContainer` as it is in this PR is only there to fetch the data. The new container would only have the job of managing selections/visibility of the Simulation.